.github/workflows/build.yml

```
@@ -18,17 +18,18 @@ jobs:
     tests:
         runs-on: ubuntu-latest
 
-        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, wkhtmltopdf: ${{ matrix.wkhtmltopdf }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, wkhtmltopdf: ${{ matrix.wkhtmltopdf }}, State Machine Adapter ${{ matrix.state_machine_adapter }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
-                symfony: ["^5.4", "^6.0"]
-                sylius: ["~1.11.4", "~1.12.0"]
+                php: ["8.1", "8.2", "8.3"]
+                symfony: ["^5.4.21", "^6.4"]
+                sylius: ["~1.12.0", "~1.13.x-dev",]
                 node: ["16.x"]
                 mysql: ["5.7", "8.0"]
```

To reduce the amount of builds, and because MySQL 5.7 is not supported anymore and we drop its support in Sylius itself, I would be for:
```suggestion
                mysql: ["8.0"]
```